clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/servergroup/EcsServerGroupController.java

```diff
@@ -0,0 +1,80 @@
+package com.netflix.spinnaker.clouddriver.ecs.controllers.servergroup;
+
+import com.amazonaws.services.ecs.AmazonECS;
+import com.amazonaws.services.ecs.model.DescribeServicesRequest;
+import com.amazonaws.services.ecs.model.DescribeServicesResult;
+import com.amazonaws.services.ecs.model.ServiceEvent;
+import com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider;
+import com.netflix.spinnaker.clouddriver.aws.security.NetflixAmazonCredentials;
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.ServiceCacheClient;
+import com.netflix.spinnaker.clouddriver.ecs.cache.model.Service;
+import com.netflix.spinnaker.clouddriver.ecs.model.EcsServerGroupEvent;
+import com.netflix.spinnaker.clouddriver.ecs.security.NetflixECSCredentials;
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/applications/{application}/{account}/{serverGroupName}")
+public class EcsServerGroupController {
+
+  private final AccountCredentialsProvider accountCredentialsProvider;
+
+  private final AmazonClientProvider amazonClientProvider;
+
+  private final ServiceCacheClient serviceCacheClient;
+
+  private final ServerGroupEventStatusConverter judge;
+
+  @Autowired
+  public EcsServerGroupController(AccountCredentialsProvider accountCredentialsProvider, AmazonClientProvider amazonClientProvider, ServiceCacheClient serviceCacheClient, ServerGroupEventStatusConverter judge) {
+    this.accountCredentialsProvider = accountCredentialsProvider;
+    this.amazonClientProvider = amazonClientProvider;
+    this.serviceCacheClient = serviceCacheClient;
+    this.judge = judge;
```

Comment:
Should update this member name to reflect the class name change as well.

Reply:
Done